plugins/LadspaEffect/caps/basics.h

```diff
@@ -76,7 +79,7 @@
 
 #define MIN_GAIN .000001 /* -120 dB */
 
-/* smallest non-denormal 32 bit IEEE float is 1.18�10-38 */
+/* smallest non-denormal 32 bit IEEE float is 1.18�10-38 */
```

Comment:
This unicode appears broken to me.
![image](https://github.com/LMMS/lmms/assets/2244945/2709848d-8fa7-41dd-8550-92b287fe5bba)

Reply:
Does this have any effect?

Reply:
![image](https://github.com/LMMS/lmms/assets/2244945/6f1849b2-7208-4efb-8db4-0ebbab58a310)
Why was this changed anyway? Here's the before:
![image](https://github.com/LMMS/lmms/assets/2244945/e1b95f24-99bf-4900-af99-355c4fe0ce4e)

Reply:
If this needs to be changed, we can use ASCII text instead: `1.18*10^38`

Reply:
I didn't change anything, maybe vscode did. Does that happen?

Reply:
> I didn't change anything, maybe vscode did. Does that happen?

Major guess here, but maybe you saved the file in a different encoding?

Reply:
fixed